src/main/scala/org/phenoscape/kb/EQForGene.scala

```diff
@@ -56,9 +57,9 @@ object EQForGene {
   }
 
   def annotationsQuery(geneIRI: IRI): Query = {
-    select_distinct('annotation) from "http://kb.phenoscape.org/" where bgp(
-      t('annotation, rdfType, Vocab.AnnotatedPhenotype),
-      t('annotation, Vocab.associated_with_gene, geneIRI))
+    select_distinct('association) from "http://kb.phenoscape.org/" where bgp(
+      t('association, rdfType, association),
+      t('association, associationHasSubject, geneIRI))
```

Comment:
For future-proofing, please check the `association` predicate here (has_phenotype).